### PR TITLE
Feature/router linking

### LINKED ISSUE #8: Integrate App Component to house network connection with Graph QL

### LINKED ISSUE #25: Begin mock data manipulation in HotTopics
set state and begin passing down info to children components

### LINKED ISSUE #26: Use tags to grab data from the mocking API call

### PR DESCRIPTION
**Pull request by Neil-B-Linton**
Hi Team, Please review this PR. 

## Description of Changes:
* set up dependencies for GraphQL
* set up dependencies for Apollo
* set up mock server to grab endpoints 
* removed React.StrictMode
* using HotTopics as container for an array of topics
* Add Topics Files (js, css, css map, sass) 
* routing setup for news articles
* created env file and queries file

## Put an 'X' next to all that apply
New Test: []
New Feature:[x]
Bug Fix: [x]

## Before You Submit Ensure
All Tests Pass: [x]
Full Coverage: [x]
Runs Locally: [x]

Thanks! 

closes #8 closes #25 closes #26

### CODE REVIEW

File: src/components/HotTopics/HotTopics.js [hunk 1]
```diff
@@ -17,60 +16,6 @@ const HotTopics = () => {
...
-//     <Link to="/newsStories/environment" onClick={event => findCurrentTag(event)} className='topics'>Environment</Link>
-//     <Link to="/newsStories/reproductive-rights" onClick={event => findCurrentTag(event)} className='topics'>Reproductive Rights</Link>
-//     <Link to="/newsStories/education" onClick={event => findCurrentTag(event)} className='topics'>Education</Link>
-//     <Link to="/newsStories/lgbtqia+" onClick={event => findCurrentTag(event)} className='topics'>LGBTQIA+</Link>
-//     <Link to="/newsStories/domestic-violence" onClick={event => findCurrentTag(event)} className='topics'>Domestic Violence</Link>
-//     <Link to="/newsStories/free-and-open-computing" onClick={event => findCurrentTag(event)} className='topics'>Free & Open Computing</Link>
-//     <Link to="/newsStories/gun-conrol" onClick={event => findCurrentTag(event)} className='topics'>Gun Control</Link>
-//     <Link to="/newsStories/all-topics" onClick={event => findCurrentTag(event)} className='topics'>All Topics</Link>
-//     <Link to="/newsStories/surprise-me" onClick={event => findCurrentTag(event)} className='topics'>Surprise Me!</Link>
-// </div>
```
Comment (camianderson) [inline src/components/HotTopics/HotTopics.js]: Thank you for removing the old lines of code.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,5 @@
 {
   "name": "one-small-thing-fe",
-  "homepage": "https://one-small-thing.github.io/One-Small-Thing-FE/",
```
Comment (camianderson) [inline package.json:3]: I think we have to keep this for our testing

File: src/components/App/App.js [hunk 3]
```diff
@@ -1,22 +1,36 @@
+import { ApolloClient, ApolloProvider, InMemoryCache } from "@apollo/client"
```
Comment (jordan-sullivan) [inline src/components/App/App.js:1]: Maybe move to index? -J

File: src/components/HotTopics/HotTopics.js [hunk 4]
```diff
@@ -1,24 +1,31 @@
 import React from 'react';
-import { Link } from 'react-router-dom'
+import Topics from './Topics.js';
 import './HotTopics.css';
 
 const HotTopics = () => {
+
+    const Array = [ "Environment", "LGBTQIA+", "Reproductive Rights", "Education", "Domestic Violence", "Free & Open Computing", "Gun Control", "All Topics", "Surprise Me!"]
```
Comment (jordan-sullivan) [inline src/components/HotTopics/HotTopics.js:7]: Will need to change name from "Array " to something better. -J

**Review by jordan-sullivan — COMMENTED** [review]
Great job setting up Graph QL and beginning routing. -J

File: package.json (hunk 2 above)
Comment (alexmkio) [reply package.json:3]: @camianderson @LINTONBNEIL This is required for your automatic deployment to work using GitHub pages.

**Comment by alexmkio** [comment]
Why are y'all merging PRs with unresolved conversations?

**Pull request merged**